docs/manifest.json

```diff
@@ -188,6 +188,7 @@
       "name": "google-cloud-dlp",
       "defaultService": "google/cloud/dlp",
       "versions": [
+        "v0.2.0",
```

Comment:
We typically don't change version numbers in pull requests. We make this change during the tagged release commit, at the same time as we update the CHANGELOG and add the new version to the main manifest. This is all covered in the RELEASING guide.